src/services/epg.service.ts

```diff
@@ -20,8 +20,8 @@ export type EpgChannel = {
   id: string;
   title: string;
   description: string;
-  channelLogoImage: ImageData;
-  backgroundImage: ImageData;
+  channelLogoImage: string | ImageData;
+  backgroundImage: string | ImageData;
```

Comment:
Is `ImageData` still valid here?

Reply:
We use it in EPG schedule. In case we change the format, it looks like some customers will have to update their data for it to continue working. My idea was to provide backward compatibility.

Reply:
The `ImageData` is composed in the `transformProgram` method, so we should be able to remove `ImageData` completely. The API response only contains an `image` property which we wrap in an object.

https://github.com/jwplayer/ott-web-app/blob/4d8ff39439e22f2e7d13ac7e73eb92a12758ddb3/src/services/epg.service.ts#L100